build/ci/evergreen.yml

```diff
@@ -794,6 +794,91 @@ tasks:
           MCLI_OPS_MANAGER_URL: ${mcli_ops_manager_url}
           MCLI_SERVICE: cloud-manager
           E2E_TAGS: cloudmanager,remote,replica
+  # Deploy a replica set, we need to be careful on running parallel modifications to the automation config
+  - name: ops_manager_4_4_deploy_replica_set_e2e
+    tags: ["e2e","clusters","opsmanager"]
+    depends_on:
+      - name: compile
+        variant: "code_health"
+        patch_optional: true
+    commands:
+      - func: "clone"
+      - func: "install gotestsum"
+      - func: "build"
+      - func: "deploy spawn host"
+      - func: ssh-ready
+      - func: "install ops manager"
+        vars:
+          ARCHIVE: ${ops_manager_4_4_archive}
+      - func: "set-up ops manager"
+      - func: "install automation agent"
```

Comment:
food for thought, should we have a single ops manager e2e sanity tests? installing OM takes a long time here so I'm not sure if it would be better to install and run the whole e2e against it

Reply:
yes, I don't see why we may need different tasks. Deploying a cluster doesn’t require a lot of time and we can always create a new project if one test interferences with another one.

Reply:
we may want to wait for the fix on 4.2 before doing this so that we can run something on 4.2.

Reply:
works for me, I'm not blocking on this just flagging it as something we may want to do